zendesk/doc.go

```diff
@@ -0,0 +1,24 @@
+// Package zendesk provides a client for using the Zendesk Core API.
+//
+// Usage:
+//
+//    package main
+//
+//    import (
+//      "log"
+//
+//      "github.com/MEDIGO/go-zendesk/zendesk"
+//    )
+//
+//    func main() {
+//        client, err := zendesk.NewClient("domain", "username", "password")
+//        if err != nil {
+//            log.Fatal(err)
+//        }
+//        ticket, err := client.ShowTicket(1)
+//        if err != nil {
+//            log.Fatal(err)
+//        }
+//        log.Printf("Requester ID is: %d", *ticket.RequesterID)
+//    }
```

Comment:
No. I don't think anything like it's in common use or exist. I think it's very useful to have code samples in the documentation, but as with any piece of documentation, that's the risk to pay.